content/Solutions/MigratingToTrueNAS.md

```diff
@@ -19,7 +31,7 @@ If you decide to use rsync, be aware that your filesystem must support Access Co
 
 ### SMB Migration
 
-We recommend migrating via SMB sharing if you use computers with Windows OS (or if you prefer SMB). 
+We recommend migrating via SMB sharing if you use computers with Windows OS (or if you prefer SMB).
 
 [Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag & drop files to migrate them from your current NAS storage into TrueNAS. Using SMB will be slower than using a utility like Robocopy.
```

Comment:
don't use the &, change to and or even drag-and-drop is okay. and don't use "will" keep it active voice and present tense